src/main/steps/check-your-answers/form.njk

```diff
@@ -0,0 +1,9 @@
+{% extends "common/form/form.njk"%}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block after_fields %}
```

Comment:
how comes this is hear and not in the template? i assume due to positioning?

Reply:
Nunjuck blocks do not inherit from one another, so you cannot use a block within a block and have to override them.

Reply:
but what about the template page fro check your answers, can you not override it there?

Reply:
Blocks can only be customised from their matching extended file. As the block `after_fields` is only in `common/form/form.njk` it cannot be customised in the page `template.njk`, as blocks are not customisable/inherited from imported files so has to be created as a separate file to be able to customise them.